shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -194,4 +202,24 @@ public static String nameFromContainer(String metric, int containerId) {
     public static String globalMetricName(String stringName) {
         return stringName + "_global";
     }
+
+    /**
+     * For some metrics such as OpStats, Pravega generates corresponding fail metrics automatically.
+     * This method provides a standard way to create name of fail metric for a given metric
+     *
+     * @param metricName - the metric name for which fail metric is created
```

Comment:
Normally, between the param name and the description there is a space or a series of spaces to align the descriptions (but no hyphens).